client/question/domain/FeedbackObjectFactorySpec.js

```diff
@@ -40,6 +40,19 @@ describe('FeedbackObjectFactory', function() {
     });
   });
 
+  describe('getErrorCategory', function() {
+    it('should return null', function() {
+      expect(feedback.getErrorCategory()).toBe(null);
+    });
+  });
+
+  describe('setErrorCategory', function() {
+    it('should set the _errorCategory property', function() {
+      feedback.setErrorCategory('ERROR OPERATOR');
```

Comment:
I think there should be a check on-set to ensure only categories we recognize are allowed, and that we should have a tie.constant('FEEDBACK_CATEGORIES', ...) whose value is an object that looks like:
```
   {
      SYNTAX_ERROR: 'SYNTAX_ERROR',
      ...
   }
```
so that this becomes

    feedback.setErrorCategory(FEEDBACK_CATEGORIES.ERROR_OPERATOR);

Otherwise we're repeating too many raw strings all over the codebase; it's easy for typos to come in.

Reply:
I've done this in #425 btw -- I needed this functionality.